cmd/internal/register/maps.go

```diff
@@ -99,12 +105,24 @@ func Maps(a *atlas.Atlas, maps []config.Map, providers map[string]provider.Tiler
 
```

Comment:
I would have expected the above at line 96 to have been changed to a `strings.SplitN(string(l.ProviderLayer),".",2)` (https://godoc.org/strings#SplitN) , this would be to make sure the name of the layer does not get messed up.